Add notice to verify Heltec runtime for Data Rate setting

leroyle opened the pull request:
As it stands version 1.0.0 of the Heltec libs set the default data rate to DR_5 which is not valid for US915, North America. This is a description about how to reset it.

We should update this with a link to a longer discussion of Data Rate, Spreading factor, transmit power and I guess other developer considerations,  once we have that figured out.

lthiery commented:
Could link to this for the time being

https://developer.helium.com/longfi/regional-channels

leroyle commented:
I've added a few words regarding the HTCC-AB02S board, the GPS flavor, added it to the "Select your board" section.  

And added notice on how the  app developer change change GPS satellite source. Seemed to help some of the community members get a more consistent GPS fix.

leroyle commented:
Is there something else I need to do to have this moved into the base?

kent-williams commented:
Looks good to me, going to merge. Thanks for the contribution @leroyle!
